thorlcr/thorutil/thmem.cpp

```diff
@@ -1659,8 +1660,9 @@ class CThorRowCollectorBase : public CSpillable
         spillableRows.save(*iFile, spillCompInfo, false, spillPrefixStr.str()); // saves committed rows
         spillFiles.append(new CFileOwner(iFile.getLink()));
         ++overflowCount;
-        sizeSpill += iFile->size();
-        spillCycles += spillTimer.elapsedCycles();
+        ++statOverflowCount; // NB: this is total over multiple uses of this class
```

Comment:
minor: There is a slightly inefficiency using atomic increments on these variables, but they will happen infrequently enough for it not to matter.  If they were frequent, better would be to use a RelaxedAtomic\<type> and use fastAdd() - which is not thread safe as far as updating goes, but doesn't need to be here.